src/base/init/install.c

```diff
@@ -237,12 +253,23 @@ static void call_installdos(void)
 		free(boot_dir_path);
 		return;
 	}
-	/* symlink command.com in case someone hits Shift or F5 */
-	ret = asprintf(&system_str,
-			"ln -s ../drives/d/command.com "
-			"../drives/d/kernel.sys "
-			"\"%s\"",
-			boot_dir_path);
+	switch (dos_status) {
+		case 0:
+			/* symlink command.com in case someone hits Shift or F5 */
+			ret = asprintf(&system_str,
+					"ln -s ../drives/d/command.com "
+					"../drives/d/kernel.sys "
+					"\"%s\"",
+					boot_dir_path);
+			break;
+		case 100:
+			ret = asprintf(&system_str,
+					"ln -s ../drives/d/COMMAND.COM "
```

Comment:
Hrrrmmm...
These case-sensitive games are risky.
Not all filesystems are case-sensitive, and,
for example vfat file-system IIRC doesn't
like the upper case (or maybe it is already
fixed).
Would it be possible for your install scripts
to always lower-case the file names to avoid
the magic like that, when you have command.com
for freedos and COMMAND.COM for ms-dos?